project-base/assets/js/commands/svg/optimizeSvg.js

```diff
@@ -1,33 +1,42 @@
-const glob = require('glob');
 const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
 const SVGO = require('svgo');
 
-const svgo = new SVGO();
+const writeSvg = (svgOptimizedFile, optimizedSvg) => {
+    try {
+        fs.writeFileSync(svgOptimizedFile, optimizedSvg);
+    } catch (err) {
+        console.log('ERROR: SVG icon ' + svgOptimizedFile + ' optimize failed');
```

Comment:
Is console.log right here? I thought that we do not want to have such code :)

Reply:
But it shows errors on output.